frontend/packages/employee-frontend/src/api/attendances.ts

```diff
@@ -220,20 +221,23 @@ function deserializeAttendanceResponse(
   {
     return {
       unit: data.unit,
-      children: data.children.sort(compareFirstNames).map((attendanceChild) => {
-        return {
-          ...attendanceChild,
-          attendance: attendanceChild.attendance
-            ? {
-                ...attendanceChild.attendance,
-                arrived: new Date(attendanceChild.attendance.arrived),
-                departed: attendanceChild.attendance.departed
-                  ? new Date(attendanceChild.attendance.departed)
-                  : null
-              }
-            : null
-        }
-      })
+      children: data.children
+        .sort((a, b) => compareByProperty(a, b, 'lastName'))
+        .sort((a, b) => compareByProperty(a, b, 'firstName'))
```

Comment:
fwiw alternative option would be `_.sortBy(data.children, ['firstName', 'lastName']).map( ...`